### PR TITLE
V3 multisampling support

### PR DESCRIPTION
**Pull request by newnon**
support for multisampling on most platforms

### CODE REVIEW

File: cocos/platform/ios/CCGLViewImpl-ios.h [hunk 1]
```diff
@@ -56,6 +56,12 @@ class CC_DLL GLViewImpl : public GLView
     static void convertAttrs();
     static void* _pixelFormat;
     static int _depthFormat;
+<<<<<<< HEAD
+=======
+    static int _multisamplingCount;
+    
+    void screenSizeChanged(int newWidth, int newHeight);
+>>>>>>> 58803da... multisampling support on all platforms(except WP)
```
Comment (minggo) [inline cocos/platform/ios/CCGLViewImpl-ios.h]: Please fix the conflict.

**Comment by newnon** [comment]
@minggo fixed

**Comment by minggo** [comment]
thanks

File: cocos/platform/CCGLView.cpp [hunk 2]
```diff
@@ -91,7 +91,7 @@ namespace {
 }
 
 //default context attributions are set as follows
-GLContextAttrs GLView::_glContextAttrs = {8, 8, 8, 8, 24, 8};
+GLContextAttrs GLView::_glContextAttrs = {5, 6, 5, 0, 16, 0, 0};
```
Comment (Dimon4eg) [inline cocos/platform/CCGLView.cpp]: Why default attributes changed?

**Comment by newnon** [comment]
I've ported this pr from old version. Fixed

**Comment by Dimon4eg** [comment]
Nice, thank you!

**Pull request merged**